<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>767</width>
    <height>617</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="fondo">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>761</width>
      <height>591</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap resource="fondo.qrc">:/fondo/fondo.jpg</pixmap>
    </property>
   </widget>
   <widget class="QTextEdit" name="codigo_fuente">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>70</y>
      <width>701</width>
      <height>171</height>
     </rect>
    </property>
   </widget>
   <widget class="QTextEdit" name="consola">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>320</y>
      <width>701</width>
      <height>201</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>767</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuOpciones">
    <property name="title">
     <string>Opciones</string>
    </property>
    <addaction name="actionImportar"/>
    <addaction name="actionAnalizar"/>
    <addaction name="separator"/>
    <addaction name="actionLimpiar"/>
   </widget>
   <addaction name="menuOpciones"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionImportar">
   <property name="text">
    <string>Importar</string>
   </property>
  </action>
  <action name="actionAnalizar">
   <property name="text">
    <string>Analizar</string>
   </property>
  </action>
  <action name="actionLimpiar">
   <property name="text">
    <string>Limpiar</string>
   </property>
  </action>
 </widget>
 <resources>
  <include location="fondo.qrc"/>
 </resources>
 <connections/>
</ui>
